frameworks/hdfs/tests/config.py

```diff
@@ -74,7 +75,23 @@ def run_hdfs_command(svc_name, command):
     return rc, output
 
 
-def check_healthy(count=DEFAULT_TASK_COUNT):
-    sdk_plan.wait_for_completed_deployment(PACKAGE_NAME, timeout_seconds=25 * 60)
-    sdk_plan.wait_for_completed_recovery(PACKAGE_NAME, timeout_seconds=25 * 60)
-    sdk_tasks.check_running(PACKAGE_NAME, count)
+def check_healthy(service_name=PACKAGE_NAME, count=DEFAULT_TASK_COUNT, recovery_expected=False):
+    sdk_plan.wait_for_completed_deployment(service_name, timeout_seconds=25 * 60)
+    if recovery_expected:
+        # TODO(elezar): See INFINITY-2109 where we need to better handle recovery health checks
+        sdk_plan.wait_for_kicked_off_recovery(service_name, timeout_seconds=25 * 60)
+    sdk_plan.wait_for_completed_recovery(service_name, timeout_seconds=25 * 60)
+    sdk_tasks.check_running(service_name, count)
+
+
+def expect_recovery(service_name=PACKAGE_NAME):
+    # TODO(elezar, nima) check_healthy also check for complete deployment, and this should not
+    # affect the state of recovery.
+    check_healthy(service_name=service_name, count=DEFAULT_TASK_COUNT, recovery_expected=True)
+
+
+def get_pod_type_instances(pod_type_prefix, service_name=FOLDERED_SERVICE_NAME):
+    pod_types = sdk_cmd.convert_output_to_list(
```

Comment:
Ok, that isn't entirely clear. I have just tested here:
```
python3 -c "import ast; ast.literal_eval('foo\nbar')"
```
and get the output:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/ast.py", line 48, in literal_eval
    node_or_string = parse(node_or_string, mode='eval')
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/ast.py", line 35, in parse
    return compile(source, filename, mode, PyCF_ONLY_AST)
  File "<unknown>", line 2
    bar
      ^
SyntaxError: unexpected EOF while parsing
```
Am I missing something?